Composer/plugins/sample-ui-plugin/src/node/index.ts

```diff
@@ -5,7 +5,7 @@
 function initialize(registration) {
   const plugin = {
     customDescription: 'Publish using custom UI',
-    hasView: true /** we have custom UI to host */,
+    bundleId: 'publish' /** we have custom UI to host */,
```

Comment:
I like this a lot better 👍

Reply:
This is to support multiple publish plugins (which I snuck into this PR).